src/components/transactions/TransactionFees.vue

```diff
@@ -0,0 +1,50 @@
+<template>
```

Comment:
This component is not only about fess. How about calling it TransactionMetaData?

Reply:
I was trying to avoid MetaData, because:

a) To me it seems lazy to label something as meta data.
b) I dislike it.

_But_, that's exactly what this is, and I can't think of anything better either 😄 I'll change it.